TombEngine/Game/control/los.cpp

```diff
@@ -133,34 +133,26 @@ bool GetTargetOnLOS(GameVector* origin, GameVector* target, bool drawTarget, boo
 
 					if (item->ObjectNumber < ID_SHOOT_SWITCH1 || item->ObjectNumber > ID_SHOOT_SWITCH4)
 					{
-						if ((Objects[item->ObjectNumber].explodableMeshbits & ShatterItem.bit) &&
```

Comment:
Why did you remove these brackets? This potentially may change precedence of operations.

Reply:
added brackets and tested again. It works also with the brackets.